packages/loader/container-loader/src/container.ts

```diff
@@ -579,6 +579,16 @@ export class Container
 		return this._lifecycleState === "disposing" || this._lifecycleState === "disposed";
 	}
 
+	/**
+	 * The error that caused the container to close or dispose.
+	 *
+	 * @remarks If the container is closed and then disposed, both with errors given, this will expose the close error only.
+	 */
+	public get closedWithError(): ICriticalContainerError | undefined {
+		return this._criticalError;
+	}
+	private _criticalError?: ICriticalContainerError;
```

Comment:
can you update the name of the variable too.

Reply:
Oops thanks.  Find/Replace mistake

Reply:
this isn't resolved yet? should it be?

Reply:
Just pushed all my updates.